### DISCUSSION TITLE
Does this work with Bunny Stream?

### DISCUSSION THREAD

**Discussion by Firsh** [cookpete/react-player#1908] [Q&A]
The closest to a controllable player I could come was this: https://bunny.net/blog/introducing-player-js-support-for-bunny-stream-advanced-player-control-and-monitoring-api/
But we currently use react-player with Vimeo, but the latter is bufffering too much and was looking for a better hosting for videos. I'm not sure if content hosted at them can be accessed in the selfhosted "file" mode of react-player.

**Comment by orbachar** [comment]
any update on this? did you end up using react-player with Bunny?
